src/articles.ts

```diff
@@ -8,12 +8,10 @@ type ArticleResult = {
 
 export class Articles {
     private dispatch: Dispatch;
-    private util: Util;

```

Comment:
Made Util use static methods (they are private anyway and not used outside). There seem to be no reason for extra allocations and repeating code.